libs/api/domains/national-registry/src/lib/soap/nationalRegistryApi.ts

```diff
@@ -13,6 +13,9 @@ import {
   GetViewFamilyDto,
 } from './dto'
 import { FamilyMember, User, FamilyRelation } from '../types'
+import { Gender } from '../types/gender.enum'
+import { MaritalStatus } from '../types/maritalStatus.enum'
```

Comment:
Tidy the imports by importing from the index file (like the import above from `../types`.